libsrc/telestrat/print.s

```diff
@@ -0,0 +1,16 @@
+; jede jede@oric.org 2017-01-22
+
+; void print (char * str);
+    .export         _print
+    .import         popax
+    .importzp       tmp1
+    .include        "telestrat.inc"
+
+.proc   _print
+    stx tmp1
+    ldy tmp1
+    BRK_TELEMON XWSTR0
```

Comment:
As far as I understand this is the same as a `write()` to fd 1. In that case please remove it. cc65 C libraries are not supposed to duplicate functionality already present through standard APIs. Just check out the other targets. All of them have some nice straight-forward ROM entry point for "just" printing a char on the screen, but non of them has a `print()` C function.